src/domain/flow-system/services/src/messages/flow_message_consumers.rs

```diff
@@ -9,6 +9,9 @@
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
+pub const MESSAGE_CONSUMER_KAMU_FLOW_TRIGGER_SERVICE: &str =
+    "dev.kamu.domain.flow-system.FlowTriggerService";
+
 pub const MESSAGE_CONSUMER_KAMU_FLOW_CONFIGURATION_SERVICE: &str =
     "dev.kamu.domain.flow-system.FlowConfigurationService";
 
```

Comment:
Consider if these events are still needed. If no, remove them, and eliminate consumption records within the migration.